distributions/aws/examples/rds-s3/README.md

```diff
@@ -77,20 +96,23 @@ Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aw
           minioServiceHost=s3.amazonaws.com
           minioServiceRegion=us-west-2
           ```
-    1. Configure `apps/pipeline/upstream/env/aws/secret.env` file with your RDS database username and password that were configured when following the steps in Create RDS Instance.
-        - For example, if your username is `admin` and your password is `Kubefl0w` then your `secret.env` file should look like:
+    1. Configure an AWS secret with your RDS database username and password that were configured when following the steps in Create RDS Instance.
         - **Note:** These are the default values for database credentials in cloudformation template for creating the RDS instance, change these according to the values you used
         - ```
           username=admin
           password=Kubefl0w
```

Comment:
do we still need this here? should not this be in secret manager ?

Reply:
Yes it should be in secret manager, I left it there so the user knows the proper keyname, am open to removing it.